app/auth/callback/adgangsplatformen/route.ts

```diff
@@ -16,7 +16,8 @@ export async function GET() {
 
   // We could not retrieve the user token.
   // So we redirect to the frontpage without setting the session.
```

Comment:
We don't redirect to the front page here right? Let's update this comment if that's the case.

Reply:
👍